packages/shared/lib/core/wallet/actions/sendOutput.ts

```diff
@@ -11,6 +13,10 @@ export async function sendOutput(output: Output): Promise<void> {
         const account = get(selectedAccount)
         const transaction = await account.sendOutputs([output], DEFAULT_TRANSACTION_OPTIONS)
         // Reset transaction details state, since the transaction has been sent
+        if (output.type === OUTPUT_TYPE_NFT) {
+            updateNftInAllAccountNfts(account.index, output.nftId, { isSpendable: false })
```

Comment:
this wont work when sending a newly minted output because `output.nftId` will be `0x000...`.
therefore we would need to get the nftId with the `getNftId` utils function. 

but because this one requires the output id and `output`  doesnt contain this, we should use the NFT output from the `transaction`, beacuse at this point, we know that `outputs.nftId` would contain the real value

Reply:
I cannot reproduce this error. I think we're confusing it with handling transaction inclusions, since we are responsible for creating the outputs and `prepareOutput()` sets the `output.nftId` value.

Reply:
confirmed :heavy_check_mark: